bin/bpmnlint.js

```diff
@@ -334,9 +337,10 @@ async function run() {
     init,
     version,
     config: configOverridePath,
+    'max-warnings': maxWarningsOverride,
     _: files
   } = mri(process.argv.slice(2), {
-    string: [ 'config' ],
+    string: [ 'config', 'max-warnings' ],
```

Comment:
Isn't it a number?

Reply:
We should provide a default for it and get rid of the overriding code: https://www.npmjs.com/package/mri#user-content-optionsdefault.

Reply:
> Isn't it a number?

there is no number option https://github.com/lukeed/mri/blob/master/index.d.ts#L8, `Numerical values are cast as Numbers when possible`